pkg/component/worker/kubelet.go

```diff
@@ -211,6 +210,11 @@ func (k *Kubelet) Run(ctx context.Context) error {
 			logrus.Warnf("failed to get initial kubelet config with join token: %s", err.Error())
 			return err
 		}
+		kubeletconfig, err = k.prepareLocalKubeletConfig(kubeletconfig)
+		if err != nil {
+			logrus.Warnf("failed to prepare local kubelet config: %s", err.Error())
+			return err
+		}
 		tw := templatewriter.TemplateWriter{
```

Comment:
Can't we now get rid of using `TemplateWriter` to dump the config as it's not really a template anymore?

Reply:
Fixed